source/developers-guide/csrf-protection/index.md

```diff
@@ -107,3 +107,25 @@ In some cases you might want to disable the protection for the backend or fronte
 ],
 ...
 ```
+
+## Plugin compatibility for older versions
+
+Since Shopware 5.2, your plugin needs to whitelist an action in order, for example, transfer files or show a page within an window or iframe but the interface `CSRFWhitelistAware` isn't available in versions prior to 5.2 and you receive an exception. In this case, you have to create a dummy interface which will only be loaded if the it does not exists.
```

Comment:
...within a window...
first sentence is very long.
second part of second sentence says what? :smile: